sklearn/adaboost/adaboost-test.yml

```diff
@@ -0,0 +1,10 @@
+---
+- doc: Test sample data for Adaboost regressor workflow
+  job: adaboost-job.yml
```

Comment:
You can save the job file by adding this directly here, like.

```
  job:
    reads_1:
      class: File
      location: https://zenodo.org/record/1324070/files/wt_H3K4me3_read1.fastq.gz
      filetype: fastqsanger.gz
```

Reply:
Fixed it! thanks!